packages/frontend/src/App.vue

```diff
@@ -15,7 +15,7 @@
             </router-link>
           </div>
           <div class="md-toolbar-section-end">
-            <hamburger />
+            <hamburger v-if="userFound" />
```

Comment:
Just semantics, but I would probably move the `v-if` to the encapsulating `<div>` as it serves no purpose if all its content is hidden anyway.

Reply:
The class on the div has impact on the placement of other divs in the header.

Reply:
That is a fair point :)